src/Analysis/Ast/Impl/Analyzer/Evaluation/ExpressionEval.Generics.cs

```diff
@@ -47,57 +48,61 @@ internal sealed partial class ExpressionEval {
             // rather than specific type instantiation as in List[str].
 
             IPythonType[] specificTypes;
+            var returnInstance = false;
             switch (expr) {
+                // Indexing returns type as from A[int]
                 case IndexExpression indexExpr: {
-                    // Generic[T1, T2, ...] or A[type]()
```

Comment:
Was this formatting change done for a specific reason? Seems weird to have an extra indent for just this one case, but maybe there's some C# semantic you're using that I'm missing.

Reply:
Yeah it is C# formatter bug with comments inside nested block. I'll fix that.